### ISSUE TITLE
broke image links

### ISSUE THREAD

**Issue by bcdc13** [Matthias-Wandel/imgcomp#17]
I've poked around as best I can, but I'm not able to get the images to show in the browser.  Everything else seems to be working, but images won't display in the browser.  I can open and view the images from the file browser.  Thoughts?

![2020-06-25-173220_1360x768_scrot](https://user-images.githubusercontent.com/67441954/85797576-e69f2700-b709-11ea-9f45-8216f96d1d05.png)

**Comment by Matthias-Wandel** [comment]
Look at the URL you are browsing.
Where on earth did that URL come from?  Why view.cgi/view.cgi?


------ Original Message ------

**Comment by bcdc13** [comment]
Honestly, I have no idea how that got there.  At this point I was able to get things working, but now that you brought the issues in the url I'm wondering if my troubleshooting actually fixed anything.  I ended up removing the imgcomp dir and recloning/reinstalling the software.  I think part of my issue initially was I was running the imgcomp command from within the imgcomp folder.  Not running it like ~ $ imgcomp/imgcomp which is how you say to run it in the documentation.  Seems I'm not so good at following instructions.  In any case, running in from inside imgcomp meant I needed to move the imgcomp.conf file around, then it wasn't saving the images to the correct place, etc (I ended up just causing more issues every time I would fix something).  Following the instructions the second time around fixed it.
